bootstrap.py

```diff
@@ -16,6 +16,10 @@
         print("Found 'waf', skipping download.")
         sys.exit(0)
 
+if "--no-download" in sys.argv[1:]:
+    print("Did not find {} and no download was requested.".format(WAFRELEASE))
```

Comment:
nitpick: it checks for the existence of `waf` not `waf-1.9.8`

Reply:
The idea is that it fails even if waf is there but not the right version, so I didn’t want to just say “can’t find waf”. How do you think it should be worded?

Reply:
you can just keep it like this